src/components/form-builder.js

```diff
@@ -24,14 +24,23 @@ function BuildForm(props) {
         return (
           <div key={field.name}>
             <label htmlFor={field.name}>{field.title}</label>
-            <input
-              type={field.type}
-              name={field.name}
-              placeholder={field.placeholder}
-              value={field.value}
-              onChange={field.onChange}
-              className={field.className}
-            />
+            <div className="input-group">
+              {field.name === 'price' ? (
+                <div className="input-group-prepend">
+                  <span className="input-group-text">$</span>
+                </div>
+              ) : (
+                <div>{null}</div>
```

Comment:
sorry i meant remove this div completely and replace it with null since there's nothing inside of it